cloudimg/ms_azure.py

```diff
@@ -379,9 +379,8 @@ def upload_to_container(self, image_path, container_name, object_name,
 
         # Azure can't handle compressed images on marketplaces so we need to
         # send the decompressed data to its storage account.
-        if object_name.endswith(".xz"):
+        if image_path.endswith(".xz"):
             log.info("Processing a LZMA compressed file: %s.", object_name)
```

Comment:
The logging is a bit wrong since `object_name` is the (output, decompressed) object name and not the (input, compressed) image file name.

Reply:
Good catch!

I've changed this line to use the `basename` of `image_path`, thanks a lot!

Reply:
Actually you fixed it in aws.py and didn't fix it here where I commented in ms_azure.py.